package.json

```diff
@@ -5,7 +5,7 @@
   "main": "./lib/index.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "build": "node ./node_modules/gulp/bin/gulp.js build"
+    "build": "gulp build"
```

Comment:
Not sure which version I prefer.  Not everyone is going to have gulp installed globally, so I kind of like the `node` version, since `gulp` will be installed to node_modules after running `npm install`.